api/cpp/esp-idf/slint/README.md

```diff
@@ -189,7 +189,7 @@ If your display controller expects little endian, set the `color_swap_16` field
 You see errors at application link time such as these:
 
 ```
-compiler_builtins.4c2482f45199cb1e-cgu.05:(.text.__udivdi3+0x0): multiple definition of `__udivdi3'; .../libgcc.a(_udivdi3.o): first defined here
+compiler_builtins.4c2482f45199cb1e-cgu.05:(.text.__udivdi3+0x0): multiple definition of `__udivdi3`; .../libgcc.a(_udivdi3.o): first defined here
```

Comment:
This was cut and pasted, I think there is no need to fix that.

Reply:
Reverted